doc/classes/RenderingServer.xml

```diff
@@ -149,6 +149,16 @@
 				Once finished with your RID, you will want to free the RID using the RenderingServer's [method free_rid] static method.
 			</description>
 		</method>
+		<method name="canvas_item_add_animation_slice">
+			<return type="void" />
+			<argument index="0" name="item" type="RID" />
+			<argument index="1" name="animation_length" type="float" />
+			<argument index="2" name="slice_begin" type="float" />
+			<argument index="3" name="slice_end" type="float" />
+			<argument index="4" name="offset" type="float" default="0.0" />
+			<description>
+			</description>
```

Comment:
Documentation should be added.

Reply:
Added. I copied the description for `CanvasItem.draw_animation_slice`, is that acceptable?

Reply:
Not my area to say for certain, but if it's not, someone on the Rendering team can ask for changes.